docs/CNAME

```diff
@@ -0,0 +1 @@
+checkit.club
```

Comment:
Oh I know for sure we can drop this though. I stopped paying for the domain, so no custom domain anymore

Reply:
I just removed this file, as its not needed :)